docs/custom-domain/README.md

```diff
@@ -1,105 +1,125 @@
 
-For greater isolation, we recommend that you put each certificate in its own key vault and set its access policy independently. The registry should always have access to the key vault secrets.
+For greater isolation, we recommend that you put each certificate in its own key vault and set its access policy independently. The registry should always have access to the key vault certficates.
 
-### Enhanced security with Virtual Networks
-Azure Key Vault allows you to [restrict access](https://docs.microsoft.com/azure/key-vault/key-vault-overview-vnet-service-endpoints) to specific virtual networks only. ACR custom domains are currently _not supported_ where key vault access is restricted, but this is work in progress and will be available with system managed identities only.
+### Enhanced security with virtual networks
+Azure Key Vault allows you to [restrict access](https://docs.microsoft.com/azure/key-vault/key-vault-overview-vnet-service-endpoints) to specific virtual networks only. Azure Container Registry custom domains can't currently be used where key vault access is restricted. This work is in progress and will be available with system-managed identities only.
    
 ## Prepare your DNS zone
 1. The custom registry domain must have a CNAME record with the target registry login server:\
    `container-registry.contoso.com` --> `myregistry.azurecr.io`
-2. The regional custom data domain must have a CNAME record with the target regional registry data endpoint:\
+2. Each regional custom data domain must have a CNAME record with the target regional registry data endpoint:\
    `eastus-registry-data.contoso.com` --> `myregistry.eastus.data.azurecr.io`
    - The output of the command to enable data endpoints on the registry will contain the regional data endpoint.
    
-## Contact us
-As a final step, share the following with us by creating a support ticket ([Azure Support](https://azure.microsoft.com/support/create-ticket/)):
+## Contact Azure support
+As a final step, share the following with us by creating an [Azure Support](https://azure.microsoft.com/support/create-ticket/) ticket. This information is needed to complete the custom domain configuration.
 - Custom registry domain details
-  - custom registry domain (`container-registry.contoso.com`)
-  - key vault secret ID of the corresponding TLS data
-  - client ID of the user assigned registry identity that has access to this secret (not required in case of system assigned)
-- Custom data domain details
-  - regional custom data domain (`eastus-registry-data.contoso.com`)
-  - key vault secret ID of the corresponding TLS data
-  - client ID of the user assigned registry identity that has access to this secret (not required in case of system assigned)
+  - custom registry domain name (`container-registry.contoso.com`)
+  - key vault certificate ID of the corresponding TLS data (a URI of the form `https://myvaultvault.azure.net/certificates/mycertificate/xxxxxxxxxxxxx`)
```

Comment:
This needs to remain a secret ID. Certificate IDs do not have the private key information.

Reply:
Fixed. Thanks for clarifying